### PR TITLE
now attempts to redirect user to the same page on ft.com when opting out

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: server/controllers/opt-out.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ const config = require('../config');
...
 const sendFeedback = require('../lib/feedback/send');
 const debug = require('debug')('ft-next-opt-in-out-debug');
+const url = require('url');
+
 
 module.exports = function(req, res) {
 	debug('Opt-Out Request headers=%j', req.headers);
 	metrics.count('opt-out');
-
+	let path = url.parse(req.get('Referrer') || 'https://next.ft.com/').path;
```
Comment (matthew-andrews) [inline server/controllers/opt-out.js]: Did you mean `Referer`?

File: test/server/opt-out.spec.js [hunk 2]
```diff
@@ -26,6 +26,15 @@ describe('Opt Out', function() {
 			.expect(302, done);
 	});
 
+	it('Should redirect to the same url on ft.com', function(done){
+		let path = '/content/randomuuid';
+		request(app)
+			.post('/opt-out')
+			.set('Referrer', 'http://next.ft.com' + path)
```
Comment (matthew-andrews) [inline test/server/opt-out.spec.js]: :trollface:

File: server/controllers/opt-out.js (hunk 1 above)
Comment (keirog) [inline server/controllers/opt-out.js]: (Both work in express)

Comment (matthew-andrews) [inline server/controllers/opt-out.js]: oh wow, magic

**Pull request merged**